src/main/java/uk/ac/ebi/eva/pipeline/io/readers/VcfHeaderReader.java

```diff
@@ -25,7 +25,7 @@
 import org.springframework.core.io.Resource;
 
 import uk.ac.ebi.eva.commons.models.data.VariantSourceEntity;
-
+import java.util.*;
```

Comment:
Use explicit imports following our internal coding guidelines. Can be configured in the IDE.